gsa/src/web/components/link/__tests__/externallink.js

```diff
@@ -62,6 +62,33 @@ describe('ExternalLink tests', () => {
     expect(dialog).toBeTruthy();
   });
 
+  test('should close confirmation dialog on resume click', () => {
+    window.open = jest.fn();
```

Comment:
You need to restore the open function after the test. Otherwise this may have side effects.